src/components/chips/js/contactChipsAutocompleteScope.js

```diff
@@ -0,0 +1,29 @@
+angular
+    .module('material.components.chips')
+    .directive('mdContactChipsAutocompleteScope', MdContactChipsAutocompleteScope);
+
+function MdContactChipsAutocompleteScope ($compile, $mdUtil) {
+  return {
+    restrict: 'A',
+    link:     postLink,
+    scope:    false
+  };
+
+  function postLink (scope, element, attr) {
+    // Since the autocomplete popup is appended to the body, we cannot traverse up the DOM looking
+    // for the proper controller and grabbing it's scope, so we must go WAY up to get the proper
+    // scope; create a new child scope based off of the parent, and copy the item over on changes
+    var newScope = scope.$parent.$parent.$parent.$new();
```

Comment:
This was all based off of the autocomplete's [`parentScope.js`](https://github.com/angular/material/blob/master/src/components/autocomplete/js/parentScope.js) directive which grabs the autocomplete controller which stores a reference to the parent, and I am able to use that, but I still had to do `ctrl.parent.$parent.$new()` to get it to work, and that turned out to be more code.

Can you clarify exactly what you mean by the following? I'm super-happy to test out an alternative.

> we might be able to access it directly from this scope